lib/import/hmcts_importer.rb

```diff
@@ -40,7 +40,8 @@ def self.default_organisation
     end
 
     def self.create_attachments(attachments, publication)
-      temp_directory = "/tmp/#{SecureRandom.uuid}"
+      # TODO: Is it safe to save these straight to a whilehall machine? If so, what directory?
+      temp_directory = "/tmp/hmcts_attachments_#{SecureRandom.uuid}"
```

Comment:
I think it'd be ok, but also we can use [`require 'tmpdir'; Dir.mktmpdir`](https://ruby-doc.org/stdlib-2.3.1/libdoc/tmpdir/rdoc/Dir.html) to get a definitely temp dir (and I think if we use the block form it'd be removed by ruby for us when we're done).